Resources/public/js/models/ez-contentmodel.js

```diff
@@ -216,10 +216,14 @@ YUI.add('ez-contentmodel', function (Y) {
          * @method loadLocations
          * @param {Object} options
          * @param {Object} options.api (required) the JS REST client instance
+         * @param {Object} options.location (optional) current location. If present it will be
```

Comment:
```
@param {Object} [options.location] current location. If present it will be
```
also don't indent the following line otherwise, you'll have to change it when changing any details of this one.